package Chapter2;

/**
 * @author : themb
 * Project : Java-Training-2026
 * Date : 9/29/2026
 **/
public class SwitchStatement {
    public static void main(String[] args){
        String module = "Java";
        String lecturer;

        switch(module){
            case "Java": lecturer ="Smith";
            break;
            case "VB":
            case "C#": lecturer ="Carol";
            break;
            case "Python": lecturer ="James";
            break;
            default: lecturer = "invalid module";
        }
        System.out.println("lecturer for module" + module + " is " + lecturer);
    }
}
